src/assets/toolkit/styles/components/sky.css

```diff
@@ -238,15 +268,27 @@
  * 1. At large enough screens, make the menu _always_ visible.
  * 2. Remove the bump (since it won't be above anything at this point).
  * 3. Reset the background.
- * 4. Reset the order.
+ * 4. To align right, this now needs to be last.
+ * 5. Constraining the width keeps the last three links on a second line to avoid orphans.
  */
 
 @media (--Sky-nav-expanded-viewport) {
   .Sky-nav-menu {
     display: block; /* 1 */
     padding-bottom: 0; /* 2 */
     background: transparent; /* 3 */
-    order: 0; /* 4 */
+    order: 2; /* 4 */
```

Comment:
<img width="851" alt="screen shot 2016-07-18 at 12 02 00 pm" src="https://cloud.githubusercontent.com/assets/1242871/16926863/8a7cf288-4cdf-11e6-93bc-d4cbb09a9d8b.png">

If I reset everything to `order: 0`, this is what happens (since ` sky-nav-controls` is first in the source order. Doing:
- logo: 0
- ` sky-nav-controls`: 1
- nav: 1
works fine, because it falls back to source order if two elements get the same `order`.

Buuuut, I think it is more clear when I set the order of nav to 2. I have to switch it to something greater than or equal to 1, since it starts off at -1. To me, switching it to 2 makes it more clear what order you can expect the elements to show up in?